applications/DemStructuresCouplingApplication/custom_utilities/compute_dem_face_load_utility.h

```diff
@@ -73,26 +78,27 @@ namespace Kratos {
                 for (unsigned int i = 0; i < source_conditions.size(); i++) {
                     ModelPart::ConditionsContainerType::iterator it = r_structural_skin_model_part.ConditionsBegin() + i;
                     Condition::GeometryType& geometry =  it->GetGeometry();
-                    double Element_Area = geometry.Area();
+                    double condition_area = geometry.Area();
 
                     for (unsigned int i = 0; i < geometry.size(); i++) { //talking about each of the three nodes of the condition
                         double& node_area = geometry[i].FastGetSolutionStepValue(DEM_NODAL_AREA);
-                        node_area += 0.333333333333333 * Element_Area; //TODO: ONLY FOR TRIANGLE... Generalize for 3 or 4 nodes
+                        node_area += 0.333333333333333 * condition_area; //TODO: ONLY FOR TRIANGLE... Generalize for 3 or 4 nodes
```

Comment:
Can you replace these written-out decimals into their fraction form? (there are a few more)= I am sure the compiler takes care of that...

Reply:
done.